css/css-pseudo/highlight-currentcolor-painting-properties-001-ref.html

```diff
@@ -25,13 +25,6 @@
 #textshadow-currentcolor > span {
   text-shadow: currentcolor 2px 2px;
 }
-#textfillcolor-currentcolor > span {
-  -webkit-text-fill-color: currentcolor;
-}
-#textstrokecolor-currentcolor > span {
-  -webkit-text-stroke-width: 1px;
-  -webkit-text-stroke-color: currentcolor;
-}
```

Comment:
The spec mentions `fill-color` and others at: https://drafts.csswg.org/css-pseudo/#highlight-styling
Why are we removing them?

Reply:
[‘fill-color’](https://drafts.fxtf.org/fill-stroke-3/#fill-color) and [‘stroke-color’](https://drafts.fxtf.org/fill-stroke-3/#stroke-color) are SVG-only properties that are unrelated to [‘-webkit-text-fill-color’](https://compat.spec.whatwg.org/#the-webkit-text-fill-color) and [‘-webkit-text-stroke-color’](https://compat.spec.whatwg.org/#the-webkit-text-stroke-color), even though they work roughly the same way. see also:

* https://svgwg.org/svg2-draft/painting.html#ColorProperty
* https://lists.w3.org/Archives/Public/www-style/2016Apr/0383.html

Reply:
Ok, I got confused about that. Thanks.